### DISCUSSION TITLE
Insert crystals that represent orientation like in OIM

### DISCUSSION THREAD

**Discussion by bicer23** [mtex-toolbox/mtex#2147] [Ask Anything]
Hey,

I am right now working on EBSD, but my supervisor wants MTEX do have following feature:

-Click on any point on the Plot(either IPF/Phase plot) which then outputs a crystal which represents the orientation.

I know that I can use the crystalShape function to insert crystals, but these a on a fixed point on the map. 

Best,
Feti

**Comment by kilir** [comment]
Hi, 

have alook at the input of crystalShape/plot https://mtex-toolbox.github.io/crystalShape.plot.html.
You could simply select any point on the map using ginput(), get the orientation and position and plot a crystalShape accordingly.
Hope that helps.

Cheers, Rüdiger.
